source/index.rst

```diff
@@ -29,14 +29,21 @@ Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation!
    docs/zero-to-robot/step-3/index
    docs/zero-to-robot/step-4/index
 
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+   :caption: Control System Overviews
+
+   docs/software/software-overview/control-system-software
```

Comment:
I disagree with Dalton, I think a Software Overview in the software section and a Hardware overview in the hardware section makes more sense than a whole section for two articles.

I don't feel super strongly so if there is a consensus for an overview section like this, that's fine.

Reply:
We don't actually make a solid distinction on software vs hardware in the TOC. I objected to having the overview under "Software Tools", but I'm not sure where you could put the software overview, unless "Programming Basics"

Reply:
I like the idea of the overviews being independent top level items.  I think we pulled them out of a prominent position in Zero to Robot but giving them one in the TOC makes sense to me.  I think there is a lot of other good cleanup we can do to the TOC to keep it manageable.  Lets discuss ideas on Slack.